### PR TITLE
Adds a phobia of guns

### PR DESCRIPTION
**Pull request by ArcaneDefence**
## About The Pull Request
Adds another type of phobia that encompasses guns, ammo, turrets, emitters, mech guns, mech ammo, and holsters.

## Why It's Good For The Game
Allows players to opt into more mechanics that can reinforce playing a character if they choose to.

## Changelog
:cl:
add: You can now choose to have a fear of guns if you take the phobia quirk.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
add a config that auto applies this to any connecting user who has an ip address assigned to an american ISP

**Comment by SuperNovaa41** [comment]
>Fermichem 2.5 causes runtimes  

unacceptable

**Comment by ATH1909** [comment]
> add a config that auto applies this to any connecting user who has an ip address assigned to an american ISP

*to a non-american ISP

**Comment by ishitbyabullet** [comment]
america bad europe good

**Comment by ishitbyabullet** [comment]
You misspelled mateba ("metaba") and incendiary ("incindiary").

Also this phobia would technically make you paranoid of magic since most magic weapons are considered guns.

File: code/controllers/subsystem/traumas.dm [hunk 1]
```diff
@@ -169,6 +170,11 @@ SUBSYSTEM_DEF(traumas)
 			/obj/item/clothing/head/helmet/space/freedom, /obj/item/clothing/suit/space/freedom
 		)),
 
+		"guns" = typecacheof(list(/obj/item/gun, /obj/item/ammo_casing/, /obj/machinery/porta_turret, /obj/machinery/power/emitter,
```
Comment (Fikou) [inline code/controllers/subsystem/traumas.dm:173]: split the obj item gun to energy and ballistic guns so it doesnt count wands, also you have a / after ammo_casing

File: strings/phobia.json [hunk 2]
```diff
@@ -331,5 +331,60 @@
...
+	"smg",
+	"lmg",
+	"m1911",
+	"357",
+	"45",
+	"9mm",
+	"10mm",
+	"uzi",
+	"revolver",
+	"metaba",
```
Comment (Fikou) [inline strings/phobia.json:364]:
```suggestion
	"mateba",
```

File: code/controllers/subsystem/traumas.dm (hunk 1 above)
Comment (ArcaneDefence) [reply code/controllers/subsystem/traumas.dm:173]: This feels more like an example of things being made guns when they're not actually guns.
Wands, staffs, meathooks, and bonechillers are all examples of that.

Just energy and ballistic isn't encompassing enough imo.

**Comment by Mothblocks** [comment]
your indenting on the json is fucked, but so is that entire file. need to run beautifiers on it some day

**Pull request merged**